### ISSUE TITLE
[ALPHA] Test Headwind 2.0.0

### ISSUE THREAD

**Issue by praveenperera** [heybourn/headwind#122]
[headwind-2.0.0.vsix.zip](https://github.com/heybourn/headwind/files/6361713/headwind-2.0.0.vsix.zip)

**Comment by edilsonborges** [comment]
I was experiencing a bug (sort of) that the features wasn't working in single quotes. Installed this version and now is working fine. Thanks! 😊

**Comment by petertriho** [comment]
@praveenperera I'll see if I can get a pull request going over the weekend to add github actions to build/publish the vsix

**Comment by praveenperera** [comment]
@petertriho that would be nice, but I don’t think I want all commits to master being published. I rather give some time to test it like this first and make more periodic releases. 

What do you think about keeping a production branch, the actions can build and publish off of that?

**Comment by petertriho** [comment]
Can have actions build off tags, releases or commits

**Comment by praveenperera** [comment]
That’s true, building off tags makes more sense.

**Comment by petertriho** [comment]
https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release

**Comment by praveenperera** [comment]
I will publish on Wednesday May 5th this week

**Comment by praveenperera** [comment]
Hey @ryanhhhh I don't have the permissions I need to publish the new version (I thought I did). Could you publish it? Thanks

**Comment by jayarjo** [comment]
@praveenperera is it out already?

**Comment by praveenperera** [comment]
> @praveenperera is it out already?

No, I need @ryanhhhh to release or give me some more permissions on vscode marketplace so I can do it myself

**Comment by iKlsR** [comment]
@ryanhhhh Can you take a look at this? Thanks.

**Comment by rushi** [comment]
@praveenperera What new features does your release have over this repo?

**Comment by BennaceurHichem** [comment]
@praveenperera  JIT mode doesn't work in this release 
having this: 
`        class="w-[120px] block mr-1 border-gray-300 mr-1rounded-md focus:border-indigo-500 sm:text-sm focus:ring-indigo-500"
`
block all the sorting due to `w-[120px]` class which is a JIT mode class, please let me know if there is a fix for that

**Comment by pelletencate** [comment]
@praveenperera Is @heybourn still supporting this repository? (And if so, what's the holdup here?) His ~~last commit dates from November 2020~~ last release to marketplace was in January 2021; It appears that you've been giving Rustywind its fair share of updates in the mean time, but I've been forced to pin it to v0.6.7 to keep it compatible with currently avaailable Headwind.

**Comment by rushi** [comment]
I think if @praveenperera can just release his own forked extension and publish it on the marketplace it would be good since he seems to be much more active? 

Thoughts?

**Comment by DerekLoop** [comment]
@praveenperera Hi! Any update on the JIT mode? I realize that Headwind is broken for JIT mode, but I couldn't sort the RegExp myself.

**Comment by phacks** [comment]
For anyone wondering how to install the 2.0.0 Alpha, here’s the command (to run after unzipping the file provided by @praveenperera in the original post):

```
code --install-extension headwind-2.0.0.vsix
```

If you don’t have the `code` CLI installed yet, you can follow these steps from the docs: https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line

**Comment by maxfenton** [comment]
Is the 2.0.0 alpha working for anyone with JIT classes? It does nothing for me.

**Comment by therealgilles** [comment]
To add support for tailwindcss custom values, the headwind.classRegex needs to be updated as follows (allows '[' and ']' characters):

```JSON
  "headwind.classRegex": {
    "html": "\\bclass\\s*=\\s*[\\\"\\']([_a-zA-Z0-9\\s\\-\\:\\/]+)[\\\"\\']",
    "css": "\\B@apply\\s+([_a-zA-Z0-9\\s\\-\\:\\/]+);",
    "javascript": "(?:\\bclassName\\s*=\\s*[\\\"\\']([_a-zA-Z0-9\\s\\-\\:\\/\\[\\]]+)[\\\"\\'])|(?:\\btw\\s*`([_a-zA-Z0-9\\s\\-\\:\\/\\[\\]]*)`)",
    "javascriptreact": "(?:\\bclassName\\s*=\\s*[\\\"\\']([_a-zA-Z0-9\\s\\-\\:\\/\\[\\]]+)[\\\"\\'])|(?:\\btw\\s*`([_a-zA-Z0-9\\s\\-\\:\\/\\[\\]]*)`)",
    "typescript": "(?:\\bclassName\\s*=\\s*[\\\"\\']([_a-zA-Z0-9\\s\\-\\:\\/\\[\\]]+)[\\\"\\'])|(?:\\btw\\s*`([_a-zA-Z0-9\\s\\-\\:\\/\\[\\]]*)`)",
    "typescriptreact": "(?:\\bclassName\\s*=\\s*[\\\"\\']([_a-zA-Z0-9\\s\\-\\:\\/\\[\\]]+)[\\\"\\'])|(?:\\btw\\s*`([_a-zA-Z0-9\\s\\-\\:\\/\\[\\]]*)`)"
  }
```

**Comment by bilogic** [comment]
@praveenperera 

How about turning this into Headwind2 on the marketplace? Thank you
